test/certs/platcert.cert.pem

```diff
@@ -0,0 +1,45 @@
+-----BEGIN CERTIFICATE-----
+MIIH7zCCBtegAwIBAgIMZSePbF8PMM7mbl17MA0GCSqGSIb3DQEBCwUAMCUxCzAJ
+BgNVBAYTAkZSMRYwFAYDVQQKEw1Gcm9tYWdlIENvcnAuMCIYDzIwMjUwMjAxMjI1
+ODExWhgPMjAzMDAyMDEyMjU4MTFaMHMxcTAhBgZngQUFAQEMF1RydXN0ZWQgQ29t
+cHV0aW5nIENvcnAuMA8GBmeBBQUBBAwFRU5JQUMwJQYGZ4EFBQEFDBtWWDMwMDAg
+LyBWYWN1dW0gVHViZXMgTWljcm8wFAYGZ4EFBQEGDAo4Njc1MzA5LVhOMIIBIjAN
+BgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA2Dkt1iRXTg7molaLv7s8Yo8dyeYH
++oS1LPXxpGwH3t0X1Yn+PA3OV0HK3IvA4/CXIhTHhKVk19CQMTZMv70tAom8Qexs
+52vssz1o0qVNRcgpC1lt/pMnJFyxZX9nz7c70Ad0by3Fjnn5TudRKqBMQ9Ii6kWJ
+RfzKH194DIOa+DqDQZ5nF0aaVrwScZTC/2WJZeRkLHBxSj68PRz6ovH7J7Txx0JX
+OZ6cfvPRUbwxmxuUW4cN0KDQWplx0AEHYz9sSWhLCfwTzzEWdLzTDmm1DppDrS9l
+nyRqukrk1A34pT79RNFhrxNh4o184kTtKY45Qwg/foBD08BaD1BD4cixNQIDAQAB
+o4IEyzCCBMcwagYDVR0jAQH/BGAwXoAQXRNrQxrXQeOaS0STRlr3maFApD4wPDEL
+MAkGA1UEBhMCVVMxCzAJBgNVBAgMAk9SMSAwHgYDVQQKDBdUcnVzdGVkIENvbXB1
+dGluZyBHcm91cIII4U8SIkthEicwEwYDVR0gAQH/BAkwBzAFBgMqAwQwggRCBgNV
+HQkBAf8EggQ2MIIEMjASBgVngQUCGTEJMAcGBWeBBQgCMBYGBmeBBQUBAjEMMAoG
+CCsGAQQBg7kqMBQGBWeBBQIXMQswCQIBAgIBAAIBADAcBgVngQUCETETMBEwCQIB
+AgIBAAIBAAQEfdDujTBfBgZngQUFAQMxVTBTFiFodHRwczovL2V4YW1wbGUuY29t
+L3VyaS1yZWZlcmVuY2UwCwYJYIZIAWUDBAIBAyEASSh3nE6PgDeFkHOlDsz8iAFc
+ShR2E5u0arQsc993UBYwggEfBgVngQUCEzGCARQwggEQAgECoIHMFgMzLjEKAQIK
+AQABAf+AAQCBCCsEAQYBg7xsolIWIGh0dHBzOi8vZXhhbXBsZTIuY29tL3Byb2Zp
+bGUtdXJpMAsGCWCGSAFlAwQCAQMhANWgfYjjBvBepEt1ma8hr9V/EbRBSH0I4lzn
+FEgOhsZdgwgrBAEGAYO8bKRRFh9odHRwczovL2V4YW1wbGUyLmNvbS90YXJnZXQt
+dXJpMAsGCWCGSAFlAwQCAQMhAJqbbSPpTlvycreQHmmNnUoGqzxI7SrrZ+B3Jz17
+XU/noQ0WBTE0MC0yCgECAQH/ggEEAQH/FidodHRwczovL215Y29tcGFueS5jb20v
+aXNvLWNlcnRpZmljYXRpb24wggJKBgdngQUFAQcCMYICPTCCAjmgggFAMIIBPDAO
+BgZngQUSAwIEBAMHBQEMBEFTVVMMBVFYLTgwgA5BMjA1MjUwLTAyNjgyMIECMjWC
+CCsEAQYBg7xsgwH/pDgwGgYFZ4EFEQMMETA4OjQyOjY0OjA4OjI0OjEwMBoGBWeB
+BREDDBEwODo0Mjo2NDowODoyNDoxMaVboC8wCwYJYIZIAWUDBAIBBCCamXJ6TKab
+HadVbVS5Ps2AYkXMPaBJ7YGnYt3UEehjPKEoMBekFTATMREwDwYDVQQDDAhza2Vs
+ZXRvcgIIH/6PZRJ5he4DAwDzlaZiFjBodHRwczovL2V4YW1wbGU0LmNvbS9jb21w
+b25lbnQtcGxhdGZvcm0tY2VydC11cmkwCwYJYIZIAWUDBAIBAyEA6ryKOmhFanH6
+op9MWC71AmqWOy98OPXwtmp/Z21q/CqHAQChYBYuaHR0cHM6Ly9leGFtcGxlMy5j
+b20vY29tcG9uZW50LWlkZW50aWZpZXJzLXVyaTALBglghkgBZQMEAgEDIQBPcU/H
+4k3z18m4eEDCjlONP44O43jL+dZlwS9VHY1tZaIxMBcMBmZsYXZvcgwKc3RyYXdi
+ZXJyeYABAjAWDAZmbGF2b3IMCWNob2NvbGF0ZYABAKNeFixodHRwczovL2V4YW1w
+bGUzLmNvbS9wbGF0Zm9ybS1wcm9wZXJ0aWVzLXVyaTALBglghkgBZQMEAgEDIQA2
+ryRSwX9okZ+6vCLVEIeSpQZYFps1D4ixrwJHGUUhBzANBgkqhkiG9w0BAQsFAAOC
+AQEAfprck81kYqHsxfGiF0lokRpFY8VikQtHgpwLMnAWGPwNLe+F8HJKPteku4JD
+Uyp9Wgnhk4xmsBT9rlJgo2uvRzedCYfacbnOUhrZugVuTxOqkDnRScIvH7FMRTfC
+J3Npvk/mcCi0x8FrKEkLrfwroJCqwo+yhIdqaL39nKMkg9fyQqpC1KDidouvlSIg
+RG1fQ6GlMCHcjFkkv03zPeyZhhjhRhPT+HvCpxQPHaqBOmIAc5boiYTrh0aMykjd
+xcn2MuvnWynaHsZMotGQCpNzN7fBJcovNd1q/DGeWGuvx1RlwY69CuQJqDJLPIrz
+bVSZNLDtWheZFn+gJ9XOVEXNrQ==
+-----END CERTIFICATE-----
```

Comment:
shouldn't there be a test to read this in and print it out somewhere?

Reply:
Yep! There will be. This is just a draft PR at the moment.